dataset_config/1000_genomes/README.md

```diff
@@ -11,9 +11,12 @@ of the [data explorer indexers](https://github.com/DataBiosphere/data-explorer-i
 Ensure the elastic search index is already running at `http://localhost:9200`.
 
 ```
-virtualenv ~/virtualenv/dump_index
-source ~/virtualenv/dump_index/bin/activate
+virtualenv ~/virtualenv/elasticsearch
```

Comment:
These instructions are in dump_index.py. Instead of reproducing here, let's just say:

For convenience, this repo reads the index from a JSON file. This way, one can
quickly try out Data Explorer without having to set up and run any
of the [data explorer indexers](https://github.com/DataBiosphere/data-explorer-indexers). Use [dump_index.py](https://github.com/DataBiosphere/data-explorer/blob/master/util/dump_index.py) to update the JSON file.